demo/src/app/pages/examples/examples/custom-edit-view-examples.html

```diff
@@ -0,0 +1,25 @@
+<h2>Customized edit and view cells examples</h2>
```

Comment:
Please, for file naming use angular style guide. For example: `custom-edit-view-examples.component.ts`, `custom-edit-view-examples.component.html`, `custom-edit-view-examples.component.scss`, `custom-edit-view-examples.service.ts`.